syntax = "proto3";

message StartSyncC2S {
    
}